src/citrine/resources/condition_template.py

```diff
@@ -12,7 +12,25 @@
 
 
 class ConditionTemplate(DataConcepts, Resource['ConditionTemplate'], TaurusConditionTemplate):
-    """A condition template."""
+    """
+    A condition template.
+
+    Parameters
+    ----------
+    name: str
+        The name of the condition template.
+    bounds: BaseBounds
+        Bounds circumscribe the values that are valid according to this condition template.
+    description: str, optional
+        A long-form description of the condition template.
+    uids: Map[str, str], optional
+        A collection of unique identifiers, each a key-value pair. The key is the "scope"
+        and the value is the identifier. The scope "id" is reserved for the internal Citrine ID,
+        which will always be a uuid4.
+    tags: List[str], optional
+        A set of tags. Tags can be used for filtering.
```

Comment:
Do we want to include a "Formatted with :: to indicate hierarchy" or some such?

Reply:
My plan was to link to the taurus docs, which have a thorough discussion of tags. I'll do that once the taurus docs are published.